redisdb/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 4.5.2 / 2022-08-05
 
 * [Fixed] Dependency updates. See [#12653](https://github.com/DataDog/integrations-core/pull/12653).
+* [Fixed] Fix the latency measurement to avoid being impacted by the agent load. See [#12505](https://github.com/DataDog/integrations-core/pull/12505).
```

Comment:
If you have a better way to explain this, hit me!

Reply:
```suggestion
* [Fixed] Fix latency measurement to correctly exclude agent load. See [#12505](https://github.com/DataDog/integrations-core/pull/12505).
```